frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -191,6 +193,14 @@ const ManageSchedulePage = ({
     select: (data) => data.global_schedule,
   });
 
+  let selectedTeamId: number;
+
+  if (currentTeam) {
+    selectedTeamId = currentTeam.id;
+  } else {
+    selectedTeamId = teamId || 0;
+  }
+
```

Comment:
This could be shortened to: 

`const selectedTeamId = currentTeam?.id ? currentTeam.id : teamId || 0;`